sdk/communication/Azure.Communication.Calling.Server/src/Models/Events/PlayAudioResultEvent.Serialization.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.Communication.Calling.Server
+{
+    public partial class PlayAudioResultEvent
+    {
+        /// <summary>
+        /// Deserialize <see cref="PlayAudioResultEvent"/> event.
+        /// </summary>
+        /// <param name="content">The json content</param>
+        /// <returns>The new <see cref="PlayAudioResultEvent"/> object.</returns>
+        public static PlayAudioResultEvent Deserialize(string content)
+        {
+            using var document = JsonDocument.Parse(content);
+            JsonElement element = document.RootElement;
+
+            Optional<ResultInfo> resultInfo = default;
+            Optional<string> operationContext = default;
+            Optional<OperationStatus> status = default;
+            Optional<string> callLegId = default;
+            foreach (var property in element.EnumerateObject())
```

Comment:
Same

Reply:
It's defined in PMA. That's why they are C# format namespaces.